FrontDesk/src/BlazorShared/Models/EmptyBaseRequest.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace BlazorShared.Models;
+
+public class EmptyBaseRequest : BaseMessage
+{
+  public string _ { get; set; }
```

Comment:
This `_` property is a dummy property to create a Empty-like request. Since some endpoints use the `CorrelationId()` from BaseRequest, FastEndpoints EmptyRequest can't be directly used.

If this dummy property isn't used, FastEndpoints throws:

_System.NotSupportedException: Only request DTOs with publicly accessible properties are supported for request binding. Offending type: [BlazorShared.Models.AppointmentType.ListAppointmentTypeRequest]
   at FastEndpoints.RequestBinder`1..cctor()_